File: Keepr.client/src/models/Vault.js
export class Vault{
  constructor(data){
    this.id = data.id
    this.creatorId = data.creatorId
    this.name = data.name
    this.description = data.description
    this.img = data.img
    this.isPrivate = data.isPrivate
    this.visits = data.visits
    this.creator = data.creator
  }
}

File: Keepr/Models/Vault.cs
namespace Keepr.Models;
public class Vault
{
  public int Id { get; set; }
  public string CreatorId { get; set; }
  public string Name { get; set; }
  public string Description { get; set; }
  public string Img { get; set; }
  public bool? isPrivate { get; set; }
  public int Visits { get; set; }
  public Account Creator { get; set; }
}